### PR TITLE
Log admin api mutations

### PR DESCRIPTION
**Pull request by tung2744**
ref https://github.com/authgear/authgear-server/pull/3041

### CODE REVIEW

File: migrations/audit/20230420194938-add_audit_log_index.sql [hunk 1]
```diff
@@ -0,0 +1,6 @@
+-- +migrate Up
+CREATE INDEX _audit_log_idx_app_id_activity_type_created_at
+  ON _audit_log (app_id, activity_type, created_at);
+
+-- +migrate Down
```
Comment (louischan-oursky) [inline migrations/audit/20230420194938-add_audit_log_index.sql:5]: The name does not match.

File: migrations/audit/20230420194938-add_audit_log_index.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+-- +migrate Up
+CREATE INDEX _audit_log_idx_app_id_activity_type_created_at
+  ON _audit_log (app_id, activity_type, created_at);
```
Comment (louischan-oursky) [inline migrations/audit/20230420194938-add_audit_log_index.sql:3]: https://www.postgresql.org/docs/current/indexes-ordering.html

`created_at DESC`

**Review by louischan-oursky — COMMENTED** [review]
Prefix all event related to mutation with `admin_api.mutation.`, e.g. `admin_api.mutation.anonymize_user.executed`

**Comment by tung2744** [comment]
@louischan-oursky Updated and rebased, thanks! New changes start from [48c4c68](https://github.com/authgear/authgear-server/pull/3047/commits/48c4c684322800ff823760e2da4d852af520456c)

**Comment by louischan-oursky** [comment]
Please rebase.

**Comment by tung2744** [comment]
@louischan-oursky Rebased, thx!

**Pull request merged**